content/en/internal/monitoring/service-info/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Service information"
+linkTitle: "Service info"
+weight: 100
+type: "docs"
+draft: true
+description: >
+    Technical details of EGI Service Monitoring
+---
+
+## Identity card
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                                | Value                                                                          |
+|-----------------------------------------|--------------------------------------------------------------------------------|
+| Name                                    |                                                                                |
+| Description                             |                                                                                |
+| URL                                     |                                                                                |
+| Support Email                           |                                                                                |
+| [Helpdesk](../../helpdesk) Support Unit | EGI Services and Service components -> ?                                       |
```

Comment:
Here the value is part of the first column.

Reply:
I do not understand. 
What value does whatever is in the cell on line 21 first column add? Why is there? Can we get rid of it?

Reply:
Sorry I was wrong, in fact the value is an example/placeholder: the parent category for the SU is "EGI Services and Service Components" then there is the name of the service-specific SU:
![Screenshot 2021-08-13 at 17 54 03](https://user-images.githubusercontent.com/451417/129386080-5dbd28a3-0ea5-4780-b818-79ac1750e099.png)
I'm fine with whatever you fell will be the best here.

Reply:
Updating it to reflect what you are showing in the above picture. It's just for the other services but GOCDB, someone needs to fill in these tables (and I don't know how to). Maybe @enolfc can?

Reply:
So you could find most of the content from the relevant wiki pages, like https://wiki.egi.eu/wiki/GOCDB, https://wiki.egi.eu/wiki/ARGO,  or others services from the tools menu.
It's also something that could be done later when the documentation of those services is migrated.
I mainly created the entry for the Configuration Database as a template.